examples/sns-sqs-lambda/Handler.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using Epsagon.Dotnet.Lambda;
+using Amazon.SimpleNotificationService;
+using Amazon.SimpleNotificationService.Model;
+using Amazon.Lambda.SNSEvents;
+using Amazon.Lambda.Core;
+
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.Json.JsonSerializer))]
+
+namespace AwsDotnetCsharp
+{
+    public class SNSSender : LambdaHandler<Request, Response>
+    {
+        public override Response HandlerFunction(Request request, ILambdaContext context)
```

Comment:
marking the function with `async` and changing the return type to `Task<Response>` will solve this issue